src/inversion_scripts/imi_preview.py

```diff
@@ -226,6 +226,10 @@ def imi_preview(
 
     plt.rcParams.update({"font.size": 18})
 
+    # Adjust plot mask if global inversion
+    if not config["isRegional"]:
```

Comment:
Instead of doing it this way, what do you think of having the logical in-line, since we still use the mask below?

Something like:
```
    plot_field(
        ax,
        prior_kgkm2h,
        cmap=cc.cm.linear_kryw_5_100_c67_r,
        plot_type="pcolormesh",
        vmin=0,
        vmax=14,
        lon_bounds=None,
        lat_bounds=None,
        levels=21,
        title="Prior emissions",
        point_sources=get_point_source_coordinates(config),
        cbar_label="Emissions (kg km$^{-2}$ h$^{-1}$)",
        mask=mask if config["isRegional"] else None,
        only_ROI=False,
    )
```

Reply:
Cool, fixed in https://github.com/geoschem/integrated_methane_inversion/pull/250/commits/d96ee0d6a658189cd2f8bb209aca2e3fc077c332